cmd/verify.go

```diff
@@ -0,0 +1,73 @@
+package cmd
```

Comment:
Can you make sure there is a test for this command line option?

Reply:
As discussed , I have created a verify_test.go for testing the verify command line option.